webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -21,6 +21,7 @@ const AnsibleHostDetail = ({
       {response?.id && (
         <>
           <Tabs
+            className="host-details-tabs"
```

Comment:
it is quite generic className, give we are not scoping the css, should we give it bit more specific name (perhaps `ansible-host-details-tabs`?